packages/core/src/lib/features/FeatureManager.ts

```diff
@@ -56,12 +56,7 @@ export class FeatureManager {
    */
   constructor(twaManifest: TwaManifest, log: Log = new ConsoleLog('FeatureManager')) {
     if (twaManifest.features.locationDelegation?.enabled) {
-      if (twaManifest.alphaDependencies?.enabled) {
-        this.addFeature(new LocationDelegationFeature());
-      } else {
-        log.error('Skipping LocationDelegationFeature. '+
-            'Enable alphaDependencies to add LocationDelegationFeature.');
-      }
+      this.addFeature(new LocationDelegationFeature());
```

Comment:
We can get rid of the Play Billing check below too, right?

Reply:
Technically, billing is bringing in an alpha dependency, so I thought it made sense to leave this in to make it clear developers are aware?